examples/.bazelversion

```diff
@@ -0,0 +1 @@
+3.7.1
```

Comment:
Should this be `4.0.0` to match the other file?

Reply:
Ah yeah, good catch

Reply:
Updated